plugins/poll/config/locales/server.en.yml

```diff
@@ -57,6 +57,7 @@ en:
     post_is_deleted: "Cannot act on a deleted post."
 
     user_cant_post_in_topic: "You can't vote because you can't post in this topic."
+    user_cant_vote: "You can't vote because your trust level is too low."
```

Comment:
Could we change this to say something like "You need to be at least XYZ to be able to vote on this poll"?